docs/developers/overview.md

```diff
@@ -0,0 +1,32 @@
+<!--
```

Comment:
you'll need to add this file to the developers' section in `docs/.vuepress/config.js` in order for it to appear in the docs. FYI you can also run them locally using 

```bash
make docs-serve
```